data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -204,28 +204,28 @@
 
         [message]
             speaker=Moreth
-            message= _ "You’ll never take this village! I’ve seen your kind kill too many townsmen, and I’ll defend them to the death!"
+            message= _ "You’ll never take this village! I'm sick of watching your kind ransack our homes and killing the townsmen! I’ll defend them to the death if I must!"
         [/message]
 
         [message]
             speaker=Deoran
             image=portraits/deoran-mad.png
-            message= _ "My kind? I am a soldier of Wesnoth, a messenger of King Haldric, and the new commander of the South Guard! What do you mean, <i>“my kind”</i>?"
+            message= _ "My kind? I am a knight of Wesnoth and a messenger of the king Haldric! What do you mean, <i>'my kind'</i>? And where is the South Guard?"
         [/message]
 
         [message]
             speaker=Moreth
-            message= _ "Sorry, sir! I thought you were one of the bandits who have been ransacking the towns and killing the villagers. What are you doing here? Sir Gerrick and I were the last of the South Guard..."
+            message= _ "My deepest apologies, sir! I thought you were another one of the bandits. What are you doing here? Sir Gerrick and I are the last of the South Guard."
         [/message]
 
         [message]
             speaker=Deoran
-            message= _ "I have been sent to take command of the South Guard and set things right. We must arm the villagers and drive off the bandits before they can do any more harm!"
+            message= _ "What? I was sent here to question Sir Loris, but it seems that the circumstances here are more dire than I had thought. We must arm the villagers and drive off the bandits before they can do any more harm!"
         [/message]
 
         [message]
             speaker=Moreth
-            message= _ "Someone should head to the river fort! Sir Gerrick will want to know that reinforcements have arrived!"
+            message= _ "Then we must head to the river fort! Sir Gerrick is guarding our remaining armaments there!"
```

Comment:
"the armory" seems smoother than "our remaining armaments" to me. Your opinions may vary.